packages/@uppy/provider-views/package.json

```diff
@@ -24,6 +27,15 @@
     "type": "git",
     "url": "git+https://github.com/transloadit/uppy.git"
   },
+  "exports": {
+    ".": "./lib/index.js",
+    "./lib/GooglePicker/googlePicker.js": "./lib/GooglePicker/googlePicker.js",
+    "./lib/GooglePicker/icons.js": "./lib/GooglePicker/icons.js",
```

Comment:
Why are these separate? And if we must, why would we add `/lib` to the path?

Reply:
Added the used functions/types to index.js.
How should I approach this? Should I add all the functions and types exported from googlePicker.js to index.ts, or just the ones used in other packages?

Since export maps apply to all consumers, we might need to include even those functions that aren’t necessarily intended for direct use by end users, but could be needed internally by other plugins.

Reply:
>Since export maps apply to all consumers, we might need to include even those functions that aren’t necessarily intended for direct use by end users, but could be needed internally by other plugins.

@Murderlon @mifi any thoughts on this ?